CHANGES.md

```diff
@@ -1,3 +1,32 @@
+## edge-19.8.6
+
+* CLI
+  * Fixed missing `clusterDomain` in render RenderTapOutputProfile
+  * Added LINKERD2_PROXY_DESTINATION_GET_SUFFIXES env variable (thanks @arminbuerkle!)
+  * Merged the CLI 'installValues' type with Helm 'Values' type 
+    * Renamed template-values.go
+    * Defined new constructor of charts.Values type
+    * Moved all Helm values related code to the pkg/charts package
+    * Bumped dependency
+    * Used '/' in filepath to remain compatible with VFS requirement
+    * Added unit test to verify Helm YAML output
+    * Added unit test for Helm YAML validation (HA)
+* Controller 
+  * Added requirement for go 1.12.9 for controller builds
```

Comment:
i'd move this to internal, and note it applies to the whole codebase, beyond just the controller, something like
```
Require go 1.12.9 to address HTTP/2 vulnerabilities
```